docs/source/conf.py

```diff
@@ -102,8 +102,7 @@
 html_theme = 'default'
 
 on_rtd = os.environ.get('READTHEDOCS', None) == 'True'
-
-if not on_rtd:  # only import and set the theme if we're building docs locally
+if (not on_rtd) or True:  # only import and set the theme if we're building docs locally
```

Comment:
@paulocheque if your are using `or True` as condition, you can remove the condition at all, as this will always be `true`

Reply:
Absolitely, I was just testing if this was causing the layout problem in RtD, thanks